projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,38 @@
+import { standardUser } from '../../sample-data/shared-users';
+import { generateMail, randomString } from '../user';
+import { clickHamburger } from '../checkout-flow';
+
+export function loginRegisteredUser() {
+  standardUser.registrationData.email = generateMail(randomString(), true);
+  cy.requireLoggedIn(standardUser);
+  cy.reload();
+}
+
+export function openMyAccountDropDown(){
+  cy.visit('/');
+  cy.wait(5000);
+  cy.onMobile(() => {
+    clickHamburger();
+  });
+  cy.get('.accNavComponent button').click();
```

Comment:
Shouldn't this be a sort of If/Else? Either you use the Click Hamburger function or the MyAccount button

Reply:
Nope, clicking the hamburger is just an additional action required for the Mobile view.